ios/LocalNetworkPrivacy/Permission-LocalNetworkPrivacy.podspec

```diff
@@ -0,0 +1,20 @@
+require 'json'
+package = JSON.parse(File.read('../../package.json'))
+
+Pod::Spec.new do |s|
+  s.name                      = "Permission-LocalNetworkPrivacy"
+  s.dependency                  "RNPermissions"
+
+  s.version                   = package["version"]
+  s.license                   = package["license"]
+  s.summary                   = package["description"]
+  s.authors                   = package["author"]
+  s.homepage                  = package["homepage"]
+
+  s.ios.deployment_target     = "10.0"
+  s.tvos.deployment_target    = "11.0"
```

Comment:
I have not personally checked, but have you checked all APIs are actually available on ios and tvos? out of curiosity, are there any restrictions for ipados / macCatalyst or macos? (just thinking cross-platform, though it's not a strict requirement or anything)

Reply:
Haven't checked yet, just based the podspec on one of the others.

Reply:
According to the docs of [NetService](https://developer.apple.com/documentation/foundation/nsnetservice) it is available on all platforms - but it has been deprecated with no obvious replacement.

Reply:
Oh that's unfortunate. And you're right, no real guidance. You probably found this as well but the only thing I found was the lower level stuff here https://developer.apple.com/library/archive/samplecode/DNSSDObjects/Introduction/Intro.html#//apple_ref/doc/uid/DTS40011371-Intro-DontLinkElementID_2 - everything links to it